RestaurantManager/Pages/Settings/SettingsPage.xaml.cs

```diff
@@ -0,0 +1,62 @@
+using RestaurantManager.Utility;
+using Xamarin.Forms;
+using Xamarin.Forms.Xaml;
+
+namespace RestaurantManager.Pages.Settings
+{
+    [XamlCompilation(XamlCompilationOptions.Compile)]
+    public partial class SettingsPage : ContentPage
+    {
+        public SettingsPage()
+        {
+            InitializeComponent();
+            switch (Utility.Settings.Theme)
+            {
+                case 0:
+                    RadioButtonSystem.IsChecked = true;
```

Comment:
We need to check the theme based on user preference. Maybe it's information we should keep as a part of our User entity?

Reply:
It's per device so saving it in the user wouldn't be smart?